### DISCUSSION TITLE
04. Mesurer l'impact environnemental

### DISCUSSION THREAD

**Discussion by hrenaud** [cnumr/best-practices-wordpress#20] [Bonnes pratiques] [answered]
@dNicolle @Yanngautreau 

Je ne sais pas qui est en charge de cette fiche entre vous deux, mais comme vous avez fait des commits dessus tous les deux, je ping les deux 🤷‍♂️

Il mais il manque la famille dans les métadatas.

**Comment by dNicolle** [comment, marked as answer]
Hello,
C'est @Yanngautreau qui a écrit la fiche.
@Yanngautreau tu t'occupes de mettre un scope ?
